### DISCUSSION TITLE
Two icons at the same time

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#20437] [Q&A] [answered]
I guess this is wrong, this is previous update (11.09.2022). It is currently impossible to check if there is such a problem.
![image](https://user-images.githubusercontent.com/59089206/192035495-1aaa596e-5527-446d-a953-ccbad08b78b9.png)

**Comment by Gabixel** [comment, marked as answer]
This was fixed in the latest update if I'm not mistaken.
I reported another problem on Discord today, and this was how the notification showed up as I updated to the hotfix:
![image](https://user-images.githubusercontent.com/43073074/192035819-040b1083-3a8f-4982-83f2-a1467aab6136.png)
